v23.1/show-backup.md

```diff
@@ -190,6 +190,34 @@ movr          | public             | vehicles                   | table       |
 . . .
 ~~~
 
+### Show a locality-aware backup
+
+To view a [locality-aware backup](take-and-restore-locality-aware-backups.html), pass locality-aware backup URIs to `SHOW BACKUP`:
+
+{% include_cached new-in.html version="v23.1.0" %}
```

Comment:
See above comment re note box here and version tag.

As we move into release work, we mainly want to use the "New-in" tag for new features.